packages/app/src/DBSearchPage.tsx

```diff
@@ -981,25 +981,34 @@ function DBSearchPage() {
 
   const isTabVisible = useDocumentVisibility();
 
+  // For live tail updates, we need to track if any rows are expanded
+  const [hasExpandedRows, setHasExpandedRows] = useState(false);
+  const [collapseAllRows, setCollapseAllRows] = useState(false);
+
   useLiveUpdate({
     isLive,
     interval: 1000 * 60 * 15,
     refreshFrequency: 4000,
     onTimeRangeSelect,
-    pause: isAnyQueryFetching || !queryReady || !isTabVisible,
+    pause:
+      isAnyQueryFetching || !queryReady || !isTabVisible || hasExpandedRows,
```

Comment:
pausing live tail on expansion feels weird (time picker still shows live when it's no longer live). should we kick the user out of live tail like we do on scroll?

this will also eliminate the state as well, as we should be able to trigger this as a callback